src/azure-cli/azure/cli/command_modules/keyvault/vendored_sdks/azure_keyvault_securitydomain/_serialization.py

```diff
@@ -931,13 +928,6 @@ def serialize_unicode(cls, data):
         except AttributeError:
             pass
 
-        try:
-            if isinstance(data, unicode):  # type: ignore
-                # Don't change it, JSON and XML ElementTree are totally able
-                # to serialize correctly u'' strings
-                return data
-        except NameError:
-            return str(data)
         return str(data)
```

Comment:
Isn't `data` already a `str` per L925? @evelyn-ys 

```suggestion
        return data
```

Reply:
I actually fully copied all the files from SDK, not sure why here's a `str()`, but I prefer not modifying files under `vendored_sdks`. BTW, this `azure-keyvault-securitydomain` vendored sdk will be removed after SDK team publish to pypi